src/sdkV2.ts

```diff
@@ -88,18 +93,27 @@ export default class CegaEvmSDKV2 {
     );
   }
 
+  async loadTreasury(): Promise<ethers.Contract> {
+    return new ethers.Contract(
+      this._treasuryAddress,
+      TreasuryAbi.abi,
+      this._signer || this._provider,
+    );
+  }
+
   async dcsGetProduct(productId: ethers.BigNumberish) {
     const cegaEntry = await this.loadCegaEntry();
-    return cegaEntry.getDCSProduct(productId);
+    return cegaEntry.dcsGetProduct(productId);
   }
 
   async dcsSetIsDepositQueueOpen(
+    isDepositQueueOpen: boolean,
```

Comment:
nit: move this below productId

think it's an easier standard in the sdk that `productId` is always the first param for product related functions

then we can call it in the actual function wherever needed